internal/venice-common/src/main/java/com/linkedin/venice/pubsub/adapter/kafka/producer/SharedProducerServiceStats.java

```diff
@@ -0,0 +1,34 @@
+package com.linkedin.venice.pubsub.adapter.kafka.producer;
+
+import com.linkedin.venice.stats.AbstractVeniceStats;
+import com.linkedin.venice.stats.Gauge;
+import io.tehuti.metrics.MetricsRepository;
+import io.tehuti.metrics.Sensor;
+
+
+public class SharedProducerServiceStats extends AbstractVeniceStats {
```

Comment:
nit: Let's follow the naming pattern here and add kafka into the class name.

Reply:
Ack.